src/v2/Apps/Partner/Components/Overview/NearbyGalleriesRail.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+import { Box, BoxProps, Flex, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { NearbyGalleriesRail_partners } from "v2/__generated__/NearbyGalleriesRail_partners.graphql"
+import { NearbyGalleriesRailRendererQuery } from "v2/__generated__/NearbyGalleriesRailRendererQuery.graphql"
+import { Carousel } from "../Carousel"
+import { useSystemContext } from "v2/Artsy"
+import { NearbyGalleryCardFragmentContainer } from "./NearbyGalleryCard"
+import { compact } from "lodash"
+import { NearbyGalleriesRailPlaceholder } from "./NearbyGalleriesRailPlaceholder"
+
+interface NearbyGalleriesRailProps extends BoxProps {
+  partners: NearbyGalleriesRail_partners
+  city: string | null
+}
+
+const NearbyGalleriesRail: React.FC<NearbyGalleriesRailProps> = ({
+  partners,
+  city,
+  ...rest
+}) => {
+  if (!partners || partners.length === 0) {
+    return null
+  }
+
+  return (
+    <Box {...rest}>
+      <Flex mb={4} justifyContent="space-between" alignItems="center">
+        <Text variant="title">Nearby Galleries</Text>
+      </Flex>
+
+      <Carousel itemsPerViewport={[2, 2, 3]}>
+        {partners.map(({ node }) => {
+          return (
+            <NearbyGalleryCardFragmentContainer
+              key={node?.id}
+              width={[300, "100%"]}
+              partner={node!}
+              city={city}
+            />
+          )
+        })}
+      </Carousel>
+    </Box>
+  )
+}
+
+const NearbyGalleriesRailFragmentContainer = createFragmentContainer(
+  NearbyGalleriesRail,
+  {
+    partners: graphql`
+      fragment NearbyGalleriesRail_partners on PartnerEdge
+        @relay(plural: true) {
+        node {
+          id
+          slug
+          ...NearbyGalleryCard_partner
+        }
+      }
+    `,
+  }
+)
+
+export const NearbyGalleriesRailRenderer: React.FC<
+  {
+    near: string
+  } & Omit<NearbyGalleriesRailProps, "partners">
+> = ({ near, ...rest }) => {
+  const { relayEnvironment } = useSystemContext()
+
+  return (
+    <QueryRenderer<NearbyGalleriesRailRendererQuery>
+      //  @ts-expect-error STRICT_NULL_CHECK
```

Comment:
I'm surprised we need this `//  @ts-expect-error STRICT_NULL_CHECK` here! What error do you get if you remove?

Reply:
Replaced `QueryRenderer` with `SystemQueryRenderer` to fix a problem described here: https://github.com/artsy/force/pull/7605. It also solve the problem with STRICT_NULL_CHECK